.circleci/common.sh

```diff
@@ -0,0 +1,2 @@
+MINICONDA_VER=4.5.4
```

Comment:
Mh, why is this here? 4.5 also seems to be old?

Reply:
No clue, it was added by someone in `master` at some point and gets overwritten every time.

Reply:
I just updated the PR to delete that file and add it to .gitignore.